### PR TITLE
Expand linting to bionics

### PR DESCRIPTION
**Pull request by illi-kun**
Definition of formatting rules for bionic modules and application of these rules to bionics.json.

### CODE REVIEW

File: tools/format/format.conf [hunk 1]
```diff
@@ -405,6 +405,28 @@ emit:chance
...
 
+# Bionics
+^bionic
+^bionic:id
+^bionic:type
+^bionic:name
+^bionic://
+^bionic:capacity
+^bionic:description
+^bionic:occupied_bodyparts
```
Comment (mugling) [inline tools/format/format.conf]: You might want `NOWRAP` here as well?

Comment (mugling) [inline tools/format/format.conf]: I think so, it cuts down on vertical lines a lot which is generally a good thing for readability

**Comment by illi-kun** [comment]
@mugling Sure, I'll add a couple of CBMs in ```examples``` in a moment.

**Comment by mugling** [comment]
BTW did you get `make lint` working? We need to find out what stops that working for some unix platforms?

**Comment by illi-kun** [comment]
@mugling 
I've installed ```jq```, and now ```make lint``` actually creates new file, but running it gives nothing.

Currently, I'm running ```format.pl``` directly and it's enough for my current tasks.

**Comment by mugling** [comment]
What platform @illi-kun? I'd like to try and get that build script working for all varieties of unix/macos

**Comment by illi-kun** [comment]
@mugling mint 18. I'm not an experienced user of linux so it can be fault on my side.

File: tools/format/format.conf (hunk 1 above)
Comment (illi-kun) [inline tools/format/format.conf]: I tested it and it created really long lines (for me, it decreases readability), so I'd prefer to keep it as is.

**Comment by illi-kun** [comment]
Test file was added.

**Pull request merged**